travis/script.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 if [[ "$TRAVIS_PHP_VERSION" != "hhvm" && "$TRAVIS_PHP_VERSION" != "hhvm-nightly" ]]; then
```

Comment:
PHP 7 should be in this list too given that it doesn't have xdebug installed on it.